README.md

```diff
@@ -17,3 +17,10 @@ npm install
 npm start
 ````
 
+## Configuration
+
+All configuration is injected with environment variables:
+
+PORT                   (default 1337)
+VALIDATOR_REGISTRY_API (default https://api.validators.ripple.com)
```

Comment:
Sorry, I meant: are we sure we want to set `https://api.validators.ripple.com` as the default?